src/main/java/nl/smallproject/www/techiteasy/services/TelevisionService.java

```diff
@@ -0,0 +1,42 @@
+package nl.smallproject.www.techiteasy.services;
+
+import nl.smallproject.www.techiteasy.models.Television;
+import nl.smallproject.www.techiteasy.repositories.TelevisionsRepository;
+import org.springframework.beans.BeanUtils;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+@Service
+public class TelevisionService {
+
+    private final TelevisionsRepository televisionsRepository;
+
+    public TelevisionService(TelevisionsRepository televisionsRepository) {
+        this.televisionsRepository = televisionsRepository;
+    }
+
+    public List<Television> getAllTelevisionsByBrand(String brand) {
+        if (brand != null) {
+            return televisionsRepository.findByBrand(brand);
+        }
+        return televisionsRepository.findAll();
+    }
+
+    public Television saveTelevision(Television television) {
+        return televisionsRepository.save(television);
+    }
+
+    public Television updateTelevision(@PathVariable Long id, @RequestBody Television television) {
```

Comment:
Deze annotaties zijn alleen van toepassing in de controller, niet in de service. Want als je hier de PathVariable en RequestBody uit het request gaat halen, waar heb je dan de controller nog voor nodig? Nee, dat is de taak van de controller, die doet alles wat met de communicatie te maken heeft. De Service doet alle business logica